src/components/EntryList/EntryList.jsx

```diff
@@ -0,0 +1,21 @@
+import { useEntry } from '../../context/EntryContext';
+import React from 'react';
+import Entry from '../Entry/Entry';
+
+export default function EntryList() {
+  const { entries } = useEntry();
+
```

Comment:
Grabs the entries and then maps through to display one entry using your Entry function that displays the entry h2 and p tag